addons/website_slides/static/src/js/slides_course_join.js

```diff
@@ -14,8 +14,9 @@ var CourseJoinWidget = publicWidget.Widget.extend({
         'click .o_wslides_js_course_join_link': '_onClickJoin',
     },
 
-    init: function (parent, channelId){
+    init: function (parent, channelId, quiz = false) {
```

Comment:
The preferred way of handling parameters in the widgets `init` method is passing a "options" parameter.
So the signature is always: `init: function (parent, options)`

Where options is a json object of various optional parameters.

Then you do:
```
this.channelId = options.channelId;
this.quiz = options.quiz || false;
```

Can you modify the signature and adapt the calls?
(I know it was already like this before your changes, but since you're altering it and the PR targets master, we might as well fix it).